runtime/filetype.vim

```diff
@@ -2484,6 +2484,10 @@ au BufNewFile,BufRead *.fsproj,*.fsproj.user	setf xml
 " VBPROJ files are Visual Studio.NET's XML-based Visual Basic project config files
 au BufNewFile,BufRead *.vbproj,*.vbproj.user	setf xml
 
+" Unison Language
+au BufNewFile,BufRead *.u				setf unison
+au BufNewFile,BufRead *.uu			setf unison
```

Comment:
These should probably be merged into one autocommand which is the general approach taken in the file.

`au BufNewFile,BufRead *.u,*.uu			setf unison`